src/main/drivers/timer.c

```diff
@@ -321,7 +321,7 @@ void configTimeBase(TIM_TypeDef *tim, uint16_t period, uint32_t hz)
     TIM_TimeBaseStructInit(&TIM_TimeBaseStructure);
     TIM_TimeBaseStructure.TIM_Period = (period - 1) & 0xFFFF; // AKA TIMx_ARR
 
-    // "The counter clock frequency (CK_CNT) is equal to f CK_PSC / (PSC[15:0] + 1)." - STM32F10x Reference Manual 14.4.11
+    // "The counter clock frequency (CK_CNT) is equal to f CK_PSC / (PSC[15:0] + 1)."
```

Comment:
We just want to delete the support, should we even delete the comments? I mean, this comment marks the source of the CK_CNT equations, and I don’t think it is necessary to delete it.

Reply:
OK, fair point.